kratos/processes/levelset_convection_process.h

```diff
@@ -395,6 +405,15 @@ class LevelSetConvectionProcess
 
         // Sets the convection diffusion problem settings
         SetConvectionProblemSettings();
+
+        if (mIsBfecc || mElementRequiresLevelSetGradient){
+            mpGradientCalculator = Kratos::make_unique<ComputeGradientProcessType>(
+            mrBaseModelPart,
+            *mpLevelSetVar,
+            *mpLevelSetGradientVar,
+            NODAL_AREA,             // TODO: could be set as an input
```

Comment:
```suggestion
            NODAL_AREA,
```
I think in this case we want to stick to `NODAL_AREA` as it is what we normally use for the L2 projection across Kratos.